probe-rs/src/architecture/arm/core/m0.rs

```diff
@@ -255,15 +259,29 @@ pub const REGISTERS: BasicRegisterAddresses = BasicRegisterAddresses {
     LR: CoreRegisterAddress(0b0_1110),
     XPSR: CoreRegisterAddress(0b1_0000),
 };
+*/
 
 pub const MSP: CoreRegisterAddress = CoreRegisterAddress(0b01001);
 pub const PSP: CoreRegisterAddress = CoreRegisterAddress(0b01010);
 
-#[derive(Clone)]
+const PC: RegisterDescription = RegisterDescription {
+    name: "PC",
+    kind: RegisterKind::PC,
+    address: CoreRegisterAddress(0b0_1111),
+};
+
+const XPSR: RegisterDescription = RegisterDescription {
+    name: "XPSR",
+    kind: RegisterKind::General,
+    address: CoreRegisterAddress(0b1_0000),
+};
+
 pub struct M0 {
     memory: Memory,
     session: Session,
 
+    registers: RegisterFile,
```

Comment:
Why does this have to be a field of the struct? Couldn't this be made a const/static var in the m0/m3/m4 module? I mean it's the same for each mx family, no?